### PR TITLE
PP-8441 add serviceId and live fields to Charge model

### PR DESCRIPTION
**Pull request by gidsg**
## WHAT YOU DID
Follow up to #3206 adding `serviceId` and `live` fields to Charge model. Remove some constructors which are no longer needed as we can use these fields from the model. 

## How to test

Code should compile
Tests should make sense

### Logging

- [ ] only emit log lines at ERROR level which require immediate attention from a support engineer. These will trigger a zendesk alert.

### Documentation

- [ ] Updated README.md for any of the following ?

* Introduced any new environment variables / removed existing environment variable
* Added new API / updated existing API definition

### CODE REVIEW

File: src/main/java/uk/gov/pay/connector/events/model/charge/CaptureConfirmedByGatewayNotification.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import java.time.ZonedDateTime;
 
 public class CaptureConfirmedByGatewayNotification extends PaymentEventWithoutDetails {
-    public CaptureConfirmedByGatewayNotification(String resourceExternalId, ZonedDateTime timestamp) {
-        super(resourceExternalId, timestamp);
+    public CaptureConfirmedByGatewayNotification(String serviceId, boolean live, String resourceExternalId, ZonedDateTime timestamp) {
+        super(serviceId, live,resourceExternalId, timestamp);
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/events/model/charge/CaptureConfirmedByGatewayNotification.java]: Missing a space after the comma.

File: src/test/java/uk/gov/pay/connector/it/base/ChargingITestBase.java [hunk 2]
```diff
@@ -70,6 +70,8 @@
     protected static final String EMAIL = randomAlphabetic(242) + "@example.com";
     protected static final long AMOUNT = 6234L;
 
+    protected static final String serviceId = "external-service-id";
```
Comment (alexbishop1) [inline src/test/java/uk/gov/pay/connector/it/base/ChargingITestBase.java]: Convention dictates that this variable name should be in `SCREAMING_SNAKE_CASE`.

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Looks good — just a couple of minor things.

**Comment by gidsg** [comment]
@alexbishop1 Thanks, I have addressed your comments by amending commits and force-pushing.

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**